src/block-shapes.ts

```diff
@@ -0,0 +1,329 @@
+enum BlockShapes {
+    HAT = 'hat',
+    BOOLEAN = 'boolean',
+    REPORTER = 'reporter',
+    C = 'c',
```

Comment:
It'd be great to get some comments on these shapes. `HAT`, `C`, and `CAP` may be confusing to others not familiar with common internal terms.

Reply:
We might also not want to call them `C` shaped. Maybe `BRANCH` or another option.